src/proto/grpc/testing/services.proto

```diff
@@ -66,6 +81,9 @@ service WorkerService {
   // Just return the core count - unary call
   rpc CoreCount(CoreRequest) returns (CoreResponse);
 
+  // Return a set of capabilities for this worker - unary call
+  rpc Capabilities(CapabilitiesRequest) returns (CapabilitiesResponse);
```

Comment:
I think this RPC only makes things more complicated. Just having the worker respond appropriately to RunClient or RunServer if given configuration is not supported seems sufficient   (either in terms of sending a status code - let's say INVALID_ARGUMENT and a message  or by actually sending a driver response withe more detailed explanation).

The qps driver needs to recognize that given functionality is not supported and stop the servers/clients that have been started for that scenario.

Also, we already have run_performance_tests.py script that knows which scenarios are supported by what language so this functionality really seems excessive (if I understand the intention correctly).